<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IpodConversationRecorverMainWindowsApp</class>
 <widget class="QMainWindow" name="IpodConversationRecorverMainWindowsApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>60</y>
      <width>441</width>
      <height>411</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; text-decoration: underline;&quot;&gt;Premiers pas :&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt; &lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Pour commencer a utiliser nos service, vous devez choisir un fichier de base de données qui contient tout vos message.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Elle doit etre au format SQLite et il est preferable d'utiliser le fichier fournis par un Iphone, Ipod de derniere genration (integrant l'application Message d'Apple)&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Pour avoir se fichier il existe plusieurs facon de le recupérer:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;	- Si votre Ipod/Iphone est Jailbreacké vous pouvez directement aller chercher le fichier dans le repertoire &lt;span style=&quot; font-style:italic;&quot;&gt;&amp;quot;&lt;/span&gt;&lt;span style=&quot; font-size:10pt; font-style:italic;&quot;&gt;/var/mobile/library/SMS/&amp;quot;&lt;/span&gt;. vous rapatrier le fichier en question (SMS.db) sur votre ordinteur et le passer au programme.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;	- Si vous n'avez pas Jaibreacké votre Ipod/Iphone vous devez utiliser Itune pour rapatrier le fichier en question. Faites une sauvegarde de votre Idevice puis ouvrez le fichier de savegarde Itune. Le programme se chargera par la suite d'aller cherche se fichier dans la sauvegarde.&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lableConvPres">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Conversation Presente :</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ContactList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>211</width>
      <height>491</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="ConversationView">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>531</width>
      <height>491</height>
     </rect>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Cantarell'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelContenu">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>10</y>
      <width>321</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Contenu de la conversation :</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>301</width>
      <height>80</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>291</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Progression de la recherhes d'elements</string>
     </property>
    </widget>
    <widget class="QProgressBar" name="Progress">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>261</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFichier">
    <property name="title">
     <string>Fichier</string>
    </property>
    <widget class="QMenu" name="menuOpen">
     <property name="title">
      <string>Open...</string>
     </property>
     <addaction name="actionSMS_db_from_an_Itune_Save"/>
     <addaction name="actionChoose_the_file"/>
    </widget>
    <addaction name="menuOpen"/>
    <addaction name="actionUpload_File_To_server"/>
    <addaction name="actionDownload_a_File_from_server"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuExport_Data">
     <property name="title">
      <string>Export Data</string>
     </property>
     <addaction name="actionTo_html_format"/>
     <addaction name="actionTo_Xml_format"/>
     <addaction name="action_2"/>
    </widget>
    <addaction name="menuExport_Data"/>
    <addaction name="actionPreferences"/>
   </widget>
   <widget class="QMenu" name="menuShare">
    <property name="title">
     <string>Share</string>
    </property>
    <addaction name="actionTo_our_securized_server"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionVersions"/>
    <addaction name="actionUpdates"/>
    <addaction name="actionDonate"/>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuFichier"/>
   <addaction name="menuEdit"/>
   <addaction name="menuShare"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionUpload_File_To_server">
   <property name="text">
    <string>Upload File To server...</string>
   </property>
  </action>
  <action name="actionDownload_a_File_from_server">
   <property name="text">
    <string>Download a File from server...</string>
   </property>
  </action>
  <action name="actionSMS_db_from_an_Itune_Save">
   <property name="text">
    <string>SMS.db from an Itune Save</string>
   </property>
  </action>
  <action name="actionChoose_the_file">
   <property name="text">
    <string>Choose the file...</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="text">
    <string>Preferences</string>
   </property>
  </action>
  <action name="actionTo_html_format">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>To html format</string>
   </property>
  </action>
  <action name="actionTo_Xml_format">
   <property name="text">
    <string>To Xml format</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>...*</string>
   </property>
  </action>
  <action name="actionTo_our_securized_server">
   <property name="text">
    <string>To our securized server ..</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionVersions">
   <property name="text">
    <string>Versions</string>
   </property>
  </action>
  <action name="actionUpdates">
   <property name="text">
    <string>Updates</string>
   </property>
  </action>
  <action name="actionDonate">
   <property name="text">
    <string>Donate</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
